client/src/main/java/io/pravega/client/netty/impl/ClientConnectionInboundHandler.java

```diff
@@ -66,7 +66,7 @@ public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
     }
 
     @Override
-    public void channelUnregistered(ChannelHandlerContext ctx) throws Exception {
+    public void channelInactive(ChannelHandlerContext ctx) throws Exception {
```

Comment:
I added a javadoc, with a pointer to the actual javadoc, which for whatever reason netty did not include on the parent class.